### DISCUSSION TITLE
ESD misregistration due to along track deformation

### DISCUSSION THREAD

**Discussion by fdelgadodelapuente** [isce-framework/isce2#867] [General]
Dear ISCE developers,

I am processing Sentinel-1 time series of the 2018 Kilauea eruption and interferograms that span the Mw 6.8 earthquake that occurred in the fault below the volcano have strong phase jumps between specific bursts. This is expected because a significant part of this deformation is in the NS direction, and therefore it is recorded by the double differential burst overlap interferograms.  

![Screenshot 2024-08-14 at 10 57 13](https://github.com/user-attachments/assets/146bc5b5-5f5c-4ae7-8d33-32217b604f4c)

This is clearly seen in the  burst overlap interferograms, in which the phase is not constant.

![Screenshot 2024-08-14 at 10 57 39](https://github.com/user-attachments/assets/2a6f84e2-b414-4f05-b97b-abb297a68187)

![Screenshot 2024-08-14 at 10 57 44](https://github.com/user-attachments/assets/ab8977aa-e4cf-4944-81b6-2a544810ca34)

Is it possible to somehow not consider the ESD phase of the specific swath with the issue and instead derive the azimuth misregistration from a single swath with no along track deformation?

Thanks,
Francisco

**Comment by piyushrpt** [comment]
You are welcome to contribute scripts that take a bounding box into consideration for estimating ESD. Defining an area not impacted by large deformation is probably easier in geographic coordinates that swath coordinates (you may have multiple tracks to work with in general) and you can use that to decide which burst overlaps to consider for histogram generation.
